Edit entries swiftui terms syntax (#1717)
Codecademy/docs: content/swiftui/concepts/viewmodifier/terms/bold/bold.md

  - 'paths/build-ios-apps-with-swiftui'
---

> The **`.bold()`** modifier method applies a bold style to text in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.

Reviewer: ```suggestion
The **`.bold()`** modifier method applies a bold font weight to [text](https://www.codecademy.com/resources/docs/swiftui/views/text).
```

## Syntax

```pseudo
Text("I will be bold text!")
    .bold()
```

> The `.bold()` modifier takes no parameters and returns the `Text` view with bold text.

Reviewer: ```suggestion
The `.bold()` modifier takes no parameters and returns a `Text` view with bold text.
```

## Example

> The following example creates bold text:

Reviewer: ```suggestion
The following example creates bolded text:
```
